packages/block-library/src/list/deprecated.js

```diff
@@ -76,4 +76,4 @@ const v1 = {
  *
  * See block-deprecation.md
  */
-export default [ v1 ];
+export default [ v0 ];
```

Comment:
What's version 0? :)

Reply:
just a random name for the first deprecated version :P